web-client/src/presenter/state.ts

```diff
@@ -1,6 +1,10 @@
 /* eslint-disable max-lines */
 import { FormattedPendingMotionWithWorksheet } from '@web-api/business/useCases/pendingMotion/getPendingMotionDocketEntriesForCurrentJudgeInteractor';
 import { GetCasesByStatusAndByJudgeResponse } from '@web-api/business/useCases/judgeActivityReport/getCaseWorksheetsByJudgeInteractor';
+import {
+  IDLE_LOGOUT_STATES,
+  IdleLogoutStateType,
+} from '@shared/business/entities/EntityConstants';
```

Comment:
This file is using getConstants already. However, given a recent discussion about eliminating getConstants, I imported directly from EntityConstants instead.